core/cloudflow-spark/src/main/scala/cloudflow/spark/SparkStreamletContext.scala

```diff
@@ -52,10 +52,13 @@ abstract case class SparkStreamletContext(
    * @param stream stream used to write the result of execution of the `StreamingQuery`
    * @param outPort the port used to write the result of execution of the `StreamingQuery`
    * @param outputMode the output mode used to write. Valid values Append, Update, Complete
+   * @param trigger Execution trigger (see https://dzone.com/articles/spark-trigger-options for details)
```

Comment:
I would rather point to the official documentation.

Reply:
Will do